pyproject.toml

```
@@ -101,6 +102,15 @@ ignore_errors = true
 
 [tool.black]
 line-length = 100
+target-version = ['py311']
```

Since our minimum required version is 310, let's keep it at that.
```suggestion
target-version = ['py310']
```

---

Done